content/post/053-great-calamity-circumvention-assembly-at-ipfs-camp.md

```diff
@@ -0,0 +1,71 @@
+---
+date: 2019-08-01
+url: 2019-08-01-great-calamity-circumvention-assembly-at-ipfs-camp
+tags: IPFS, IPFS Camp
+title: The Great Calamity Circumvention Assembly at IPFS Camp 2019
+author: Alan Shaw
+---
+
+> This is the story of the Deep Dive sessions at IPFS Camp 2019 as told from the perspective of a DWeb historical researcher in the far future.
+
+In 2019, the intergalactic dweb uprising began. The pillars of what had become the foundations of human civilisation started to crack under their own weight and a new empire was emerging from the shadows. An empire forged in the flames of discontent of central servers and "do-no-evil" monopoly. Peer to peer communications - the heart of the dweb - slowly formed efficient and autonomous guerrilla networks that expanded beyond the reach of the blue planet and far into the galaxy, the likes of which the old world order had never seen before.
```

Comment:
```suggestion
In 2019, the intergalactic dweb uprising began. The pillars of what had become the foundations of human civilisation started to crack under their own weight and the very beginnings of a new empire began emerging from the shadows. An empire forged in the flames of discontent of central servers and "do-no-evil" monopoly. Peer to peer communications - the heart of the dweb - slowly formed efficient and autonomous networks that expanded beyond the reach of the blue planet and far into the galaxy, the likes of which the old world order had never seen before.
```
"guerilla" has some unfortunate connotations. While I like the "rebel alliance" reference, I want to avoid some of the other synonyms and the sentence works fine without any fighter allusion.